### PR TITLE
Drop MAD support

### PR DESCRIPTION
**Pull request by lenisko**
Since it's annoying to move around code with legacy MAD support, I went through and removed it. This should make RM easier to extend.

~~So far untested~~ Would be nice to have someone to test it and give feedback.

### CODE REVIEW

**Comment by lenisko** [comment]
Briefly tested and seems to work just fine. Waiting for others.

**Comment by Mygod** [comment]
This is the worst PR ever. 😡

**Comment by Mygod** [comment]
> Well tested at this point... :)

I will never believe you. :rage:

File: server/src/models/Pokemon.js [hunk 1]
```diff
@@ -317,26 +259,13 @@ class Pokemon extends Model {
       }
     }
     // second query for pvp
-    if (!mem && queryPvp && (!isMad || reactMapHandlesPvp)) {
+    if (!mem && queryPvp && reactMapHandlesPvp) {
```
Comment (Mygod) [inline server/src/models/Pokemon.js:262]: Bruh why you so bad at logic

File: server/src/models/Pokemon.js [hunk 2]
```diff
@@ -317,26 +259,13 @@ class Pokemon extends Model {
...
-        ])
-        .andWhereBetween(isMad ? 'pokemon.longitude' : 'lon', [
-          args.minLon,
-          args.maxLon,
-        ])
-      if (isMad && listOfIds.length) {
+        .where('expire_timestamp', '>=', ts)
+        .andWhereBetween('lat', [args.minLat, args.maxLat])
+        .andWhereBetween('lon', [args.minLon, args.maxLon])
+      if (listOfIds.length) {
```
Comment (Mygod) [inline server/src/models/Pokemon.js:268]: wtf?????

**Comment by Mygod** [comment]
PR reverted. Too many bugs. You might as well do it from scratch.

**Comment by lenisko** [comment]
Yeah, thanks for revert.

What can I say, my army of testers is just bad :)

**Comment by Mygod** [comment]
Maybe be careful and not write logical bugs to begin with, so that your army wouldn't need to test all the edge cases. :rage:
